test/Interop/Cxx/class/destructors-with-temporary-values-irgen.swift

```diff
@@ -0,0 +1,16 @@
+// RUN: %target-swiftxx-frontend -emit-ir -I %S/Inputs -validate-tbd-against-ir=none %s | %FileCheck %s
+
+// TODO: make this test work on Windows
+// XFAIL: OS=windows-msvc
```

Comment:
why can't you add a match line for Windows that matches the right function name ABI? We should stop proliferating XFAIL windows in tests if there's no real need for it.

Reply:
I couldn't get the Windows name out of the CI, and I don't have a Windows machine to test this locally. If this fix is not super urgent, I'll find a Windows VM to run this.